tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_IPI.py

```diff
@@ -35,16 +42,42 @@ class TestAutomatedRecoveryFromFailedNodes(ManageTest):
     threads = []
 
     @pytest.fixture(autouse=True)
-    def teardown(self, request):
+    def teardown(self, request, nodes):
 
         def finalizer():
             worker_nodes = get_worker_nodes()
             # Removing created label on all worker nodes
             remove_label_from_worker_node(worker_nodes, label_key="dc")
             for thread in self.threads:
                 thread.join()
-            ceph_health_check()
 
+            # Start the nodes which are in poweredoff state
+            not_ready_nodes = [
```

Comment:
similar implementation exists in different tests, like:
tests.e2e.monitoring.test_monitoring_on_negative_scenarios.TestMonitoringBackedByOCS.teardown
tests/manage/z_cluster/nodes/test_nodes_maintenance.py:40

It would be better to move this fixture to conftest in order to avoid code duplication

Reply:
I actually see now that these fixtures are being moved to conftest.py with https://github.com/red-hat-storage/ocs-ci/pull/2492/files